test/unit/batching/vendor-lines/get-payment-type.test.js

```diff
@@ -0,0 +1,108 @@
+const { BPS, CS, FDMR, LumpSums, SFI, SFIP, VetVisits, IMPS, ES } = require('../../../../app/constants/schemes')
+const { getPaymentType } = require('../../../../app/batching/vendor-lines/get-payment-type')
+
+describe('get payment type tests', () => {
+  let paymentType
+
+  test('return 1 when schemeId is CS and paymentType is 1', () => {
+    paymentType = '1'
+    const result = getPaymentType(CS, paymentType)
+    expect(result).toBe(1)
+  })
+
+  test('return 0 when schemeId is CS and paymentType is 0', () => {
+    paymentType = '0'
+    const result = getPaymentType(CS, paymentType)
+    expect(result).toBe(0)
+  })
+
+  test('return 0 when schemeId is CS and paymentType is 2', () => {
+    paymentType = '0'
+    const result = getPaymentType(CS, paymentType)
+    expect(result).toBe(0)
+  })
+
+  test('return "" when schemeId is BPS and paymentType is undefined', () => {
+    paymentType = undefined
+    const result = getPaymentType(BPS, paymentType)
+    expect(result).toBe('')
+  })
+
+  test('return "" when schemeId is BPS and paymentType is "0" ', () => {
+    paymentType = '0'
+    const result = getPaymentType(BPS, paymentType)
+    expect(result).toBe('')
+  })
+
+  test('return "" when schemeId is FDMR and paymentType is undefined', () => {
+    paymentType = undefined
+    const result = getPaymentType(FDMR, paymentType)
+    expect(result).toBe('')
+  })
+
+  test('return "" when schemeId is FDMR and paymentType is "0" ', () => {
+    paymentType = '0'
+    const result = getPaymentType(FDMR, paymentType)
+    expect(result).toBe('')
+  })
+
+  test('return 0 when schemeId is LumpSums and paymentType is "0" ', () => {
+    paymentType = '0'
+    const result = getPaymentType(LumpSums, paymentType)
```

Comment:
Why is lump sums and vet visits yellow?

Reply:
Yellow?

Reply:
On GitHub they appear yellow compared to the other schemes which are blue

Reply:
So they do.  How odd.  I suspect that's just weird GitHub formatting rules.  It also shows them in grey at the top whilst the rest are blue.

Probably because these ones aren't in caps.

Reply:
Not a clue